lexicons/app/bsky/feed/describeFeedGenerator.json

```diff
@@ -0,0 +1,32 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.describeFeedGenerator",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Returns information about a given feed generator including TOS & offered feed URIs",
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["did", "feeds"],
+          "properties": {
+            "did": {"type": "string", "format": "did"},
+            "feeds": {
+              "type": "array",
+              "items": {"type": "string", "format": "at-uri"}
+            },
```

Comment:
As a matter of future-proofing, would there be any value is making this an array of objects, e.g. `[{ uri }]`?  The thought is that if we ever want to add an optional field alongside the URI we'd have a place to put it.  If that seems unlikely, this is good!

Reply:
ah yeah that makes sense

I dig 👍